ui-react/src/components/topBar/LogsCanvas.tsx

```diff
@@ -10,6 +10,8 @@ type LogsCanvasProps = {
 
 export default function LogsCanvas({ show, handleClose, lastLines }: LogsCanvasProps) {
 
+  const splitted = lastLines?.split('\r\n');
```

Comment:
Split only by `\n` for non-Windows systems.
```suggestion
  const splitted = lastLines?.split('\n');
```

Reply:
Thanks, 
done.